src/pages/TableDetail.svelte

```diff
@@ -19,22 +20,49 @@
     <col />
     <col />
     <tr>
-      <td>BigQuery Definition</td>
-      <td>
-        <a href={table.bq_definition}>
-          {table.bq_definition.split("/").slice(-1)}
-        </a>
+      <td
+        >Stable
+        <HelpHoverable
+          content={"BigQuery table containing stable ping data, rolled up once per day with additional deduplication applied as well as partitioning by sample id. Use this table for most analysis."}
+          link={"https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html#table-layout-and-naming"}
+        />
       </td>
+      <td><code>{table.stable_table}</code></td>
     </tr>
     <tr>
-      <td>Live Data</td>
+      <td
+        >Live
+        <HelpHoverable
+          content={"BigQuery table containing live ping data as it is submitted to our pipeline. Only use this table if you need results for the current (partial) day."}
+          link={"https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html#table-layout-and-naming"}
+        />
+      </td>
       <td><code>{table.live_table}</code></td>
     </tr>
     <tr>
-      <td>Stable View</td>
-      <td><code>{table.stable_table}</code></td>
+      <td
+        >Schema
```

Comment:
This seems less interesting to most people than the live/stable table information, so I moved it to the bottom.